myapp/spec/factories/task.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :task do
     name { 'sample_task' }
+    status { 'unstarted' }
```

Comment:
👍 It would be more convenient to use `Trait`.
https://github.com/brennovich/cheat-ruby-sheets/blob/master/factory_bot.md#trait